images/manageiq-base/Dockerfile

```diff
@@ -57,16 +62,28 @@ RUN yum -y install https://rpm.nodesource.com/pub_10.x/el/7/x86_64/nodesource-re
                    &&               \
     yum clean all
 
-RUN mkdir -p ${APPLIANCE_ROOT} && \
-    curl -L https://github.com/${APPLIANCE_ORG}/manageiq-appliance/tarball/${APPLIANCE_REF} | tar vxz -C ${APPLIANCE_ROOT} --strip 1
+RUN if [[ -n "$GIT_TOKEN_AVAILABLE" ]] ; then \
+      export GIT_TOKEN=$(wget -O - -q http://localhost:8000/git_token.txt) ; \
+      export GIT_HOST=${GIT_TOKEN}@${GIT_HOST} ; \
+    fi && \
+    mkdir -p ${APPLIANCE_ROOT} && \
+    curl -L https://${GIT_HOST}/${APPLIANCE_ORG}/${REPO_PREFIX}-appliance/tarball/${APPLIANCE_REF} | tar vxz -C ${APPLIANCE_ROOT} --strip 1
 
-RUN mkdir -p ${SUI_ROOT} && \
-    curl -L https://github.com/${SUI_ORG}/manageiq-ui-service/tarball/${SUI_REF} | tar vxz -C ${SUI_ROOT} --strip 1
+RUN if [[ -n "$GIT_TOKEN_AVAILABLE" ]] ; then \
+      export GIT_TOKEN=$(wget -O - -q http://localhost:8000/git_token.txt) ; \
+      export GIT_HOST=${GIT_TOKEN}@${GIT_HOST} ; \
```

Comment:
This format makes it feel like the token is a `user:pass` format rather than a single token. If that's the case maybe a better name would be `GIT_AUTH`?

Reply:
yeah i like GIT_AUTH